components/CommonHeader.tsx

```diff
@@ -0,0 +1,107 @@
+/* eslint-disable jsx-a11y/alt-text  */
+/* eslint-disable @next/next/no-img-element */
+import MenuIcon from '@mui/icons-material/Menu'
+import AppBar from '@mui/material/AppBar'
+import Box from '@mui/material/Box'
+import Button from '@mui/material/Button'
+import Container from '@mui/material/Container'
+import IconButton from '@mui/material/IconButton'
+import Menu from '@mui/material/Menu'
+import MenuItem from '@mui/material/MenuItem'
+import Toolbar from '@mui/material/Toolbar'
+import Typography from '@mui/material/Typography'
+import * as React from 'react'
+import { theme } from 'theme/theme'
+
+import OSLogo from '../assets/darkThemeLogo.svg'
+
+const pages = ['About', 'Projects', 'Partner', 'Volunteer', 'Events']
+
+const CommonHeader = () => {
+  // React states and functions for handling the hamburger menu.
+  const [anchorElNav, setAnchorElNav] = React.useState<null | HTMLElement>(null)
+
+  const handleOpenNavMenu = (event: React.MouseEvent<HTMLElement>) => {
+    setAnchorElNav(event.currentTarget)
+  }
+
+  const handleCloseNavMenu = () => {
+    setAnchorElNav(null)
+  }
+
+  return (
+    <AppBar position="static" sx={{ background: theme.palette.primary.main }}>
+      <Container maxWidth="xl">
+        <Toolbar sx={{ height: '50px' }}>
+          {/* LOGO */}
```

Comment:
The comments in the return statement are quite helpful here. I haven't worked on a project that used comments like this, is it common practice? I think the Homepage could definitly use some.

Reply:
We can make it a common practice :)